src/scripts/pages/categories.js

```diff
@@ -25,32 +25,38 @@ window.addEventListener('DOMContentLoaded', (event) => {
     $navCategoryHeaderMinfraud.setAttribute('aria-expanded', 'true');
     $navCategoryHeaderMinfraud.classList.remove('is-closed');
   } else if ($accountCategory) {
+    $body.classList.add('page-type--account');
     $navCategoryHeaderAccount.classList.add('is-open');
     $navCategoryHeaderAccount.setAttribute('aria-expanded', 'true');
     $navCategoryHeaderAccount.classList.remove('is-closed');
   } else if ($dataPrivacyCategory) {
+    $body.classList.add('page-type--data-privacy');
     $navCategoryHeaderDataPrivacy.classList.add('is-open');
     $navCategoryHeaderDataPrivacy.setAttribute('aria-expanded', 'true');
     $navCategoryHeaderDataPrivacy.classList.remove('is-closed');
   }
 
-  if ($categoryTitle.innerText === 'GeoIP2 and GeoLite2') {
-    $body.classList.add('page-type--geoip');
-    $navCategoryHeaderGeoip.classList.add('is-open');
-    $navCategoryHeaderGeoip.setAttribute('aria-expanded', 'true');
-    $navCategoryHeaderGeoip.classList.remove('is-closed');
-  } else if ($categoryTitle.innerText === 'minFraud Web Services') {
-    $body.classList.add('page-type--minfraud');
-    $navCategoryHeaderMinfraud.classList.add('is-open');
-    $navCategoryHeaderMinfraud.setAttribute('aria-expanded', 'true');
-    $navCategoryHeaderMinfraud.classList.remove('is-closed');
-  } else if ($categoryTitle.innerText === 'Account and Purchasing') {
-    $navCategoryHeaderAccount.classList.add('is-open');
-    $navCategoryHeaderAccount.setAttribute('aria-expanded', 'true');
-    $navCategoryHeaderAccount.classList.remove('is-closed');
-  } else if ($categoryTitle.innerText === 'Data Privacy') {
-    $navCategoryHeaderDataPrivacy.classList.add('is-open');
-    $navCategoryHeaderDataPrivacy.setAttribute('aria-expanded', 'true');
-    $navCategoryHeaderDataPrivacy.classList.remove('is-closed');
+  if ($categoryTitle) {
+    if ($categoryTitle.innerText === 'GeoIP2 and GeoLite2') {
+      $body.classList.add('page-type--geoip');
+      $navCategoryHeaderGeoip.classList.add('is-open');
+      $navCategoryHeaderGeoip.setAttribute('aria-expanded', 'true');
+      $navCategoryHeaderGeoip.classList.remove('is-closed');
+    } else if ($categoryTitle.innerText === 'minFraud Web Services') {
```

Comment:
I wonder if there is a way to reduce the duplicate code. I tried using or statements such as the following, but it didn't work.

```
else if ($minfraudCategory || $categoryTitle.innerText === 'minFraud Web Services')
```